source/core/CEntryPoints.cpp

```diff
@@ -159,6 +159,9 @@ VIREO_EXPORT EggShellResult EggShell_DeallocateData(TypeManagerRef tm, const Typ
 VIREO_EXPORT EggShellResult EggShell_FindValue(TypeManagerRef tm, const char* viName, const char* eltName, TypeRef* typeRefLocation, void** dataRefLocation)
 {
     TypeManagerScope scope(tm);
+    *typeRefLocation = nullptr;
```

Comment:
just making this function follow the same pattern of nulling out the pointer storage location at the start in-case we return early